src/porepy/applications/test_utils/material_constants_for_testing.py

```diff
@@ -0,0 +1,47 @@
+from porepy.applications.material_values.solid_values import granite
```

Comment:
I am OK with the centralized placement of the values for testing. Any other opinion @IvarStefansson  @keileg

Reply:
Sure!